src/main/java/pl/rejmanbeata/bakery/controller/ClientController.java

```diff
@@ -0,0 +1,69 @@
+package pl.rejmanbeata.bakery.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.rejmanbeata.bakery.model.client.Client;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/clients")
+public class ClientController {
+    private final List<Client> clientList = new ArrayList<>();
```

Comment:
I must admit, it drew my attention :) 
Out of curiosity, why are we using `List<Client` here instead of `ClientService` ...? :)